### PR TITLE
Build cron for data seed demo

### PR DESCRIPTION
**Pull request by thomtrp**
- extracts the logic of the command into a service
- builds a module for dependency injections that export this service
- runs this service into the existing command
- builds a job + cron command. Set to run at 10pm
- puts everything in a separated folder

Local output for both command and cron:
Capture d’écran 2024-02-23 à 12 21 35

### CODE REVIEW

File: packages/twenty-server/src/database/commands/crons/data-seed-demo-workspace.job.ts [hunk 1]
```diff
@@ -1,27 +1,24 @@
...
+} from 'src/database/typeorm-seeds/core';
 
-@Command({
-  name: 'workspace:seed:demo',
-  description:
-    'Seed workspace with demo data. This command is intended for development only.',
-})
-export class DataSeedDemoWorkspaceCommand extends CommandRunner {
+@Injectable()
+export class DataSeedDemoWorkspaceJob implements MessageQueueJob<undefined> {
```
Comment (charlesBochet) [inline packages/twenty-server/src/database/commands/crons/data-seed-demo-workspace.job.ts:15]: @thomtrp I think we should keep the command to seed the demo workspace manually if needed.
Also the job should be in a jobs folder

**Review by charlesBochet — COMMENTED** [review]
Thank you @thomtrp, I've left comments :)

File: packages/twenty-server/src/database/commands/data-seed-demo-workspace-command.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+import { MessageQueue } from 'src/integrations/message-queue/message-queue.constants';
+import { MessageQueueService } from 'src/integrations/message-queue/services/message-queue.service';
+
+@Command({
+  name: 'workspace:seed:demo',
+  description: 'Seed workspace with demo data. Use in development only.',
+})
+export class DataSeedDemoWorkspaceCommand extends CommandRunner {
+  constructor(
+    @Inject(MessageQueue.messagingQueue)
```
Comment (thomtrp) [inline packages/twenty-server/src/database/commands/data-seed-demo-workspace-command.ts:15]: @charlesBochet not sure about this one. Not even sure that I should trigger the job there, wdyt?

**Review by charlesBochet — APPROVED** [review]
LGTM, thank you :)

**Pull request merged**